openedx/features/discounts/applicability.py

```diff
@@ -28,6 +28,18 @@
 from common.djangoapps.track import segment
 
 
+# .. toggle_name: discounts.enable_first_purchase_discount_override
+# .. toggle_implementation: WaffleFlag
+# .. toggle_default: False
+# .. toggle_description: Waffle flag to enable the First Purchase Discount to be overriden from
+#   EDXWELCOME/BIENVENIDOAEDX 15% discount to a new code with 30% discount.
```

Comment:
I would make this comment more general to say "to a new code" -- because the new code can be configurable via a setting of some kind

Reply:
Yes, now that we decided to add the coupon code in settings make sense to not add the percentage here.